src/frontend/README-E2E.md

```diff
@@ -0,0 +1,15 @@
+# playwright
+## installation
+Run `npm i` to install the Playwright from the package.json.
+
+## running tests
+To run Playwright tests you will need to specify the config file. We have created NPM scripts to make like a bit easier:
+
+`npm run pl:staging` - runs playwright tests against staging environment in headed browser
+`npm run pl:staging:headless` - same as above but headless
+`npm run pl:staging:debug` - runs Playwright in debug mode
+`npm run pl:report` opens test results report in the browser
+
+NOTES: 
+You need to be in the `src/frontend` to run these tests
```

Comment:
this is fine for now, i'll make a follow up ticket to add these commands to the makefile.

Reply:
Yes indeed, we need a separate ticket to configure this in makefile and CI.